commands/common/create_session.py

```diff
@@ -66,7 +66,7 @@ def process_json(self, redfishConfig, url):
         JsonBuilder.addElement('main', JsonType.STRING, 'UserName', redfishConfig.get_value('username'))
         JsonBuilder.addElement('main', JsonType.STRING, 'Password', redfishConfig.get_value('password'))
 
-        link = UrlAccess.process_request(redfishConfig, UrlStatus(url), 'POST', False, json.dumps(JsonBuilder.getElement('main'), indent=4))
+        link = UrlAccess.process_request(redfishConfig, UrlStatus(url), 'POST', False, JsonBuilder.getElement('main'))
```

Comment:
@jskazinski 
I'm confused.  I thought you stated in the meeting that your additions here were adding the `pretty-print` capability with an indent.  However, I see the `indent` parameter being removed in this line's difference.
Then, as a separate example, in `core/urlAccess.py` line 193, I see you adding the `indent` parameter.
So, the way I'm reading this is that you are adding the `pretty-print` in some locations and removing it in others.
Is that correct or am did I misinterpret something?

Reply:
For the case where UrlAccess.process_request() is called, that function now calls the requests.request(method, ...) function and that function will send the json data as is, with CR & NL inserted, so the pretty print does not work as expected with the requests library and I am removing them from all UrlAccess.process_request() calls.

In core/urlAccess.py I am adding the json.dumps() call to pretty print the json data because the json data being passed in is no longer pretty printed. 

After review, this is needed when switching to the requests library and allows the json data being sent via http/https to be received correctly by the Redfish Service.